modules/tinymce/src/plugins/link/test/ts/browser/DialogSectionsTest.ts

```diff
@@ -1,140 +1,159 @@
-    );
-
-    const sCheckClassSection = (exists: boolean, value: Optional<Array<{ value: string; title: string }>>) => Log.step('TBA', 'Link: sCheckClassSection',
-      sCheckSections([
-        {
-          setting: { key: 'link_class_list', value },
-          selector: 'label:contains("Class")',
-          exists
-        }
-      ])
-    );
-
-    const sCheckLinkListSection = (exists: boolean, value: Optional<Array<{ value: string; title: string }>>) => Log.step('TBA', 'Link: sCheckLinkListSection',
-      sCheckSections([
-        {
-          setting: { key: 'link_list', value },
-          selector: 'label:contains("Link list")',
-          exists
-        }
-      ])
-    );
-
-    // NOTE: Always end with none to remove the setting from future tests
-    Pipeline.async({}, [
-      TestLinkUi.sClearHistory,
-      sCheckTargetSection(false, Optional.some(false)),
-      sCheckTargetSection(true, Optional.some(true)),
-      sCheckTargetSection(true, Optional.none()),
-
-      sCheckRelSection(true, Optional.some([
-        { title: 'a', value: 'b' },
-        { title: 'c', value: 'd' }
-      ])),
-      sCheckRelSection(false, Optional.none()),
-
-      sCheckTitleSection(false, Optional.some(false)),
-      sCheckTitleSection(true, Optional.none()),
-
-      sCheckClassSection(true, Optional.some([
-        { title: 'a', value: 'b' },
-        { title: 'c', value: 'd' }
-      ])),
-      sCheckClassSection(false, Optional.none()),
-
-      sCheckLinkListSection(true, Optional.some([
-        { title: 'a', value: 'b' },
-        { title: 'c', value: 'd' }
-      ])),
-      sCheckLinkListSection(false, Optional.none()),
-      TestLinkUi.sClearHistory
-    ], onSuccess, onFailure);
-  }, {
-    plugins: 'link',
-    toolbar: 'link',
-    theme: 'silver',
-    base_url: '/project/tinymce/js/tinymce'
-  }, success, failure);
+  const checkTitleSection = (exists: boolean, value: Optional<boolean>) => {
+    checkSections([
+      {
+        setting: { key: 'link_title', value },
+        selector: 'label:contains("Title")',
+        exists
+      }
+    ]);
+  };
+
+  const checkRelSection = (exists: boolean, value: Optional<Array<{ value: string; title: string }>>) => {
+    checkSections([
+      {
+        setting: { key: 'rel_list', value },
+        selector: 'label:contains("Rel")',
+        exists
+      }
+    ]);
+  };
+
+  const checkClassSection = (exists: boolean, value: Optional<Array<{ value: string; title: string }>>) => {
+    checkSections([
+      {
+        setting: { key: 'link_class_list', value },
+        selector: 'label:contains("Class")',
+        exists
+      }
+    ]);
+  };
+
+  const checkLinkListSection = (exists: boolean, value: Optional<Array<{ value: string; title: string }>>) => {
+    checkSections([
+      {
+        setting: { key: 'link_list', value },
+        selector: 'label:contains("Link list")',
+        exists
+      }
+    ]);
+  };
+
+  context('TBA: Check Target section', () => {
```

Comment:
Some more where we probably shouldn't have `TBA` on a context. Also unrelated, but the tests here a little weird, but I don't have any suggestions off the top of my head.